go.mod

```diff
@@ -146,3 +145,5 @@ require (
 )
 
 replace github.com/aws/amazon-sagemaker-operator-for-k8s => github.com/aws/amazon-sagemaker-operator-for-k8s v1.0.1-0.20210303003444-0fb33b1fd49d
+
+replace github.com/flyteorg/flyteplugins => github.com/flyteorg/flyteplugins v1.0.3-0.20220606160608-f19ef782f97c
```

Comment:
Just leaving a note as a reminder to upgrade to the released version

Reply:
updated it